lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -398,7 +398,7 @@ class ClassMetadataInfo implements ClassMetadata
      *
      * @var array
      *
-     * @psalm-var array<string, array{type: string, fieldName: string, columnName: string, inherited: class-string}>
+     * @psalm-var array<string, array{type: string, fieldName: string, columnName?: string, inherited?: class-string, nullable?: bool, originalClass?: class-string, originalField?: string}>
```

Comment:
I think linebreaks are ok inside psalm annotations.

Reply:
I'll do that